src/osd/OSDMap.cc

```diff
@@ -57,6 +57,7 @@ using ceph::decode;
 using ceph::encode;
 using ceph::Formatter;
 
+#define dout_context g_ceph_context
```

Comment:
eek sorry, forgot to remove this from some work I cancelled. please ignore for now.